scripts/irods/start_options.py

```diff
@@ -1,10 +1,10 @@
 def add_options(parser):
     parser.add_option('-q', '--quiet',
-                      dest='verbose', action='store_false',
+                      dest='verbose', action='store_const', const=0,
```

Comment:
i'm not familiar with store_const ... so does `-q` still do what it did?   or do we now just set a numeric value for `-v`?

Reply:
The behavior of `-q` is effectively the same.

I've changed the `verbose` variable from a boolean to an integer.
You can pass `-v` multiple times, and each time increments `verbose`, allowing for additional levels of verbosity.
`-q` just explicitly sets `verbose` to 0.

Reply:
ah, nice.   and if they're both passed, which wins?

Reply:
I've preserved the previous behavior of `-q` cancelling out any `-v` that comes before in the argument list.